docs/sources/flow/reference/stdlib/trim_space.md

```diff
@@ -0,0 +1,16 @@
+---
+aliases:
+- ../../configuration-language/standard-library/trim_space/
+title: trim_space
+---
+
+# trim_space
+
+`trim_space` removes any space characters from the start and end of a string.
```

Comment:
Can we change this to `removes any white space characters from the start and end of a string.`? My fear is people will interpret space literally.

Reply:
We could use `empty-space` (avoids color bias which is a thing we need to be aware of:  https://grafana.com/docs/writers-toolkit/write/style-guide/inclusive-writing/#avoid-charged-language ). Not a must, but should be considered here.

Reply:
By all respect, I never hear about empty-space and it could be misleading term. I understand inclusive guide, but unlike whitelist -> allowlist, empty-space is not well defined in common sense. Could result into more confusion.

I would stick with white space here, to have an clear technical documentation. But if you wish empty-space, I would change it again.

Reply:
I am generally in favor of inclusive language, but I think "white space" is called that because paper is white, and doesn't fall under typical charged language. 

I also agree that empty-space could be confusing, and I personally wouldn't know what it's referring to.